src/main/kotlin/app/revanced/patches/all/misc/versioncode/ChangeVersionCodePatch.kt

```diff
@@ -12,9 +12,9 @@ import org.w3c.dom.Element
 @Patch(
     name = "Change version code",
     description = "Changes the version code of the app. By default the highest version code is set. " +
-            "This allows older versions of an app to be installed " +
-            "if their version code is set to the same or a higher value and can stop app stores to update the app. " +
-            "This does not apply when installing with root install (mount).",
+            "If the version code is set to the same or higher value as an exisiting installation " +
+            "the app can be installed as an update. This can also prevent app stores from updating " +
+            "the app, except in the case of a root install (mount).",
```

Comment:
From my understanding, the exception of the root install is only regarding the app store issue. Without this change, this is ambiguous since it was placed in its own sentence

Reply:
No the version code cannot be changed in the root (mount) installation method
(The version code is treated as unchanged not only in the app store but also in all other apps)

Here is a brief technical explanation:

For root(mount) install method, [app signature, libraries, AndroidManifest.xml are not changed](https://github.com/ReVanced/revanced-patches-template/pull/2178#issuecomment-1552412139)

The original app's app signature, libraries, and AndroidManifest.xml are still in memory, so they are unchanged

Here are the Pros and Cons of this:

Pros
1. Since the app signature of the original app is maintained, root users will not experience playback issues without patches such as `Spoof client`
2. Because it is recognized as the same as the original app, Google cannot determine whether the user is using RVX or YouTube

Cons
1. Since AndroidManifest.xml does not change, there are restrictions on patch implementation. For example, the `Change version code` patch, which requires modification of AndroidManifest.xml, does not work
2. Since it is recognized as the same as the original app, if the user does not use the magisk detach module, automatic updates will be applied from Google PlayStore

Reply:
So the patch can't even be used to downgrade to a lower version when mounting?

Reply:
yeah it only works for non-root user